bootstrap-salt.sh

```diff
@@ -1258,24 +1258,30 @@ __check_dpkg_architecture() {
 
     __REPO_ARCH="$DPKG_ARCHITECTURE"
 
-    if [ "$DPKG_ARCHITECTURE" = "i386" ]; then
-        echoerror "$_REPO_URL likely doesn't have all required 32-bit packages for Ubuntu $DISTRO_MAJOR_VERSION (yet?)."
+    case $DPKG_ARCHITECTURE in
+        "i386")
+            error_msg="$_REPO_URL likely doesn't have all required 32-bit packages for $DISTRO_NAME $DISTRO_MAJOR_VERSION."
+            # amd64 is just a part of repository URI, 32-bit pkgs are hosted under the same location
+            __REPO_ARCH="amd64"
+            ;;
+        "amd64")
+            error_msg=""
+            ;;
+        "armhf")
+            error_msg=""
```

Comment:
Since there are no `armhf` packages for Ubuntu and Debian 7, maybe we should also check that here?